### DISCUSSION TITLE
Complete your Outreachy internship final application : Dot the I's and cross the  T's

### DISCUSSION THREAD

**Discussion by gbengaoluwadahunsi** [camicroscope/caMicroscope#887] [General]
Hello @everyone
Thanks to @birm for the guidance and feedback for the past weeks and to everyone who has contributed (and still contributing) to this project.
I do hope we are all having a great time as contributors, just so some of us don't miss out on the final necessities, Outreachy sent out an email and we are required to do the following, 

Final Steps for Applying
-------------------------

In order to be eligible to be selected as an Outreachy intern, you must:

1. Verify your time commitments are correct in your initial application.

2. Record at least one in-progress contribution you've made to an Outreachy internship project. The contribution must be recorded through the Outreachy website.

3. Create a final application for that project. The final application must be submitted through the Outreachy website.

You must create a final application for each project. If you don't create a final application for a project, you aren't eligible to be selected as an intern for that project.

Verify Your Time Commitments
----------------------------

Your approval to participate in Outreachy is based on the time commitments you listed in your initial application. You can check your listed time commitments by following these steps:

1. Log into the Outreachy website:

https://www.outreachy.org/login/

2. Check the time commitments listed on your initial application:

https://www.outreachy.org/eligibility-results/

If your time commitments are incorrect or have changed since you filled out an initial application, please contact us immediately:

https://www.outreachy.org/contact/applicant-help/

Recording Contributions
-----------------------

You will need to record your contributions for each project you're applying for.

Please record your contributions even if the contribution is not merged or accepted yet.

You can record your contributions following these steps:

After the final application deadline, you are encouraged (but not required) to continue working on contributions with your project mentor. You will be able to record or edit contributions until May 1, 2024 at 4pm UTC.

Final Applications
------------------

You will need to create a final application for each project you're applying for. Only applicants that complete a final application for that project will be eligible to be selected as an intern for that project.

You can create a final application by following these steps:

1. Log into the Outreachy website:

https://www.outreachy.org/login/

2. Find your project listed on the project selection page:

https://www.outreachy.org/apply/project-selection/

3. Click the project's title to be taken to the project details page.

4. Follow the 'create a final application' link.

5. Repeat steps 2-4 to create a final application for each project you are applying for.

The final application will ask you to fill out a timeline for the internship project. If you have trouble with this, or a mentor isn't responsive to your questions about the timeline, do the best you can. You should submit a final application by the final application deadline, even if you feel the timeline isn't complete or perfect.

Goodluck

**Comment by blessdamy** [comment]
@gbengaoluwadahunsi Thank you for the reminder

**Reply by gbengaoluwadahunsi** [reply]
You are welcome

**Comment by harshpreet14** [comment]
Hey @gbengaoluwadahunsi The outline for the project "Create a Standalone Executable" has not been mentioned in the GSOC repository of caMicroscope. So, where shall we find out the required deliverables ? @blessdamy Did you find more about it?

**Reply by gbengaoluwadahunsi** [reply]
Hello @harshpreet14 

I think  GOSC  might have her own unique set of projects, I  proceeded in picking one of the projects there for caMicroscope and developed a proposal around it.

**Reply by harshpreet14** [reply]
Thankyou @gbengaoluwadahunsi , this definitely is a good way to go about it .

**Comment by barchakuz** [comment]
@gbengaoluwadahunsi  thanks brother

**Reply by gbengaoluwadahunsi** [reply]
You are welcome

**Reply by barchakuz** [reply]
@gbengaoluwadahunsi  
Hi,I have a question. Should we leave the "merged contribution date" empty, or should we fill it in? If not, please let me know.

**Reply by gbengaoluwadahunsi** [reply]
If (your PR is merged already)
{"the date should definitely be filled in"}

else{"leave blank and update later"}


Thank you😊😊😊

**Comment by AminaIbra** [comment]
The tutorial link wont work so I can follow the direction on the contribution. I sent and email to our mentors but I have not gotten a response as of yet. I am running out of time. When did you all get your approval notification for the final project? I only received this email about 5 days ago so I am a bit worried. Any help would be appreciate it!
![image](https://github.com/camicroscope/caMicroscope/assets/165629016/d443ee4f-90cc-43ac-993b-c728cf0a7c08)

**Reply by gbengaoluwadahunsi** [reply]
Helllo, this is the link to submit your final application https://www.outreachy.org/outreachy-june-2024-internship-round/communities/camicroscope/#create-stanalone-executable-for-camicroscope

**Reply by birm** [reply]
Thanks for pointing that out. The correct link is https://camicroscope.org/contributing.html

**Reply by gbengaoluwadahunsi** [reply]
Thank you @birm

**Reply by AminaIbra** [reply]
Thank you @birm and @gbengaoluwadahunsi! I am trying to go through the link so I understand what is needed prior to the deadline but I did not get access to the correct link until this afternoon so i've been struggling and juggling around to understand how to do this correctly. I am so bummed to miss on this and wish I was notified of my initial application approval and access to the correct link at least a week ago so I can prep and contribute to the project :(

**Reply by gbengaoluwadahunsi** [reply]
oh, you meant you haven't been able to make any contribution yet?

1. Take a deep breathe and smile, it never the end and you can make it through.
2. This is what I will suggest you do
    - go through this link https://camicroscope.org/contributing.html  and hop up to any task 
    - clone the caMicroscope repo https://github.com/camicroscope/caMicroscope
    - Try to identify a task you can fix based on the contributing.html list and make a pull request
    - After making the pull request, move to add it on your outreachy contribution page
    - come up with a proposal and submit your final application
    
 3. Now that the above is done, relax and then
      - come back to the camicrosope  and read some of the discussion and try to make further contributions "because you can still make 
        contributions till May 1".
        
        
        I wish you the best!
